src/components/v5/shared/MembersAvatars/types.ts

```diff
@@ -0,0 +1,8 @@
+import { Watcher } from '~types';
+
+export interface MembersAvatarsProps<TValue extends Watcher> {
+  currentDomainId?: number;
+  maxAvatars?: number;
+  className?: string;
+  items: TValue[];
```

Comment:
Can you use UserAvatar interface here? I'm not sure if generic type is needed in this case